selfdrive/ui/qt/offroad/networking.h

```diff
@@ -12,8 +12,9 @@ class WifiItem : public QWidget {
   void setItem(const Network& n, const QPixmap &icon, bool show_forget_btn, const QPixmap &strength);
 
 signals:
-  void connectToNetwork(const Network &n);
-  void forgotNetwork(const Network &n);
+  // Cannot pass Network by reference. it may change after the signal is sent.
```

Comment:
Also fixes a bug caused by passing network by references. (introduced in  #29037)

Reply:
Can you make that fix a new PR?

Reply:
split to https://github.com/commaai/openpilot/pull/29448